BUILD_FROM_SOURCE_README.md

```diff
@@ -0,0 +1,52 @@
+# Build Instructions
+
+The base tag this release is branched from is `20.0.1`
+
+---
+## Build
+Prerequisites: Building Keycloak from source requires the following:
+* JDK 11 (or newer)
+* Maven 3.5.4 (or newer).
+
+Building Keycloak server from source and creating an image, assumes the source files from the repository https://github.com/verrazzano/keycloak from branch oracle/release/20.0.1 or a branch created from oracle/release/20.0.1, is available under base directory _verrazzano_.
```

Comment:
Maybe just mention in steps
1. Clone https://github.com/verrazzano/keycloak
2. checkout branch oracle/release/20.0.1 or a branch created from oracle/release/20.0.1
3. etc..

Reply:
Done